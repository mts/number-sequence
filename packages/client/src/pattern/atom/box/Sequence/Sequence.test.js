import sequenceDefault from './Sequence.render'

describe('<Sequence />', () => {
  describe('Snaphot', () => {
    test('must match sequenceDefault', () => {
      expect(global.renderToJSON(sequenceDefault)).toMatchSnapshot()
    })
  })
})
