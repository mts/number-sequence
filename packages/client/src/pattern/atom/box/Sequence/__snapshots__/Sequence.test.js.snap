// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sequence /> Snaphot must match sequenceDefault 1`] = `
<div
  className="px-3 border text-green border-blue"
>
  5
</div>
`;
